.github/workflows/check.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  pull_request:
+    push:
+      branches:
+        - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
```

Comment:
I've been using Node 17.x locally - is there a reason we'd want to use 16 instead? Might it be worth adding a `.nvmrc` and [configuring this action to use the version specified there](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file) so that we have a consistent Node version across local dev and CI?

Reply:
Even if you use `.nvmrc` the setup-node action won't let you use 17.x, [see here](https://github.com/actions/setup-node#supported-version-syntax). Odd versions of Node are not LTS and Node recommend not to use them in production so 16 is better here anyway.

Reply:
Aha, okay. Should we perhaps add a `.nvmrc` specifying 16.x then?

Reply:
Oh yeah, good point. I've added one now 👍